conf/app.routes

```diff
@@ -1,9 +1,9 @@
 # microservice specific routes
 
-POST        /v2/declaration                                  uk.gov.hmrc.exports.controllers.DeclarationController.create()
-GET         /v2/declaration                                  uk.gov.hmrc.exports.controllers.DeclarationController.findAll()
-GET         /v2/declaration/:id                              uk.gov.hmrc.exports.controllers.DeclarationController.findByID(id: String)
-DELETE      /v2/declaration/:id                              uk.gov.hmrc.exports.controllers.DeclarationController.deleteByID(id: String)
+POST        /v2/declarations                                 uk.gov.hmrc.exports.controllers.DeclarationController.create()
```

Comment:
:+1:

Reply:
This will need updating in the frontend application.conf